components/script/dom/eventsource.rs

```diff
@@ -482,8 +483,6 @@ impl EventSource {
             data: String::new(),
             last_event_id: String::new(),
         };
-        // TODO: use the "remote event task source", and canceller.
-        // https://html.spec.whatwg.org/multipage/#remote-event-task-source
```

Comment:
This comment is a mistake I introduced in https://github.com/servo/servo/pull/21126, the networking source is the right one here, since it's a plain fetch and not a eventsource 'task', see https://html.spec.whatwg.org/multipage/server-sent-events.html#dom-eventsource